content/docs/deploy/core.mdx

```diff
@@ -117,8 +115,7 @@ If you prefer building from source:
    git clone https://github.com/pomerium/pomerium.git $HOME/pomerium
    cd $HOME/pomerium
    ```
-2. **(Optional) Generate Local Certs**  
-   For local development, use [mkcert](https://mkcert.dev/):
+2. **(Optional) Generate Local Certs** For local development, use [mkcert](https://mkcert.dev/):
```

Comment:
changes here and above are just stripping white space at the end of the lines. I can remove these changes from the PR if you'd like.